src/assets/src/components/edit.tsx

```diff
@@ -190,6 +269,17 @@ function QueueEditor(props: QueueEditorProps) {
                         </select>
                     </div>
                 </div>
+                <div className="form-group row">
+                    <label htmlFor="allowed meeting types" className="col-md-2 col-form-label">Allowed Meeting Types:</label>
+                    <div className="col-md-6">
+                        <AllowedBackendsForm 
+                            queue={props.queue}
+                            allowed={new Set(props.queue.allowed_backends)}
+                            onChange={props.onUpdateAllowedBackends}
+                            disabled={props.disabled}
+                            backends={props.backends}/>
```

Comment:
Not absolutely sure this is where I'm talking about, but I noticed when BlueJeans isn't configured (and I guess this would be the case whenever you have one backend), the `Checkbox` is still click-able. Might be nice to flip the `disabled` switch when this happens. Probably not a super common use case to have one backend, but I found it to be confusing. Similarly, if you try to un-click the last checked backend, it throws an error, which is somewhat hard to see since it's at the top of the page. Disabling the last checkbox and adding a note that you can't go down to zero might improve the interface.

Reply:
I agree, but let's leave this one for a future release.

Reply:
You want I should open a new issue?

Reply:
https://github.com/tl-its-umich-edu/remote-office-hours-queue/issues/133